style.css

```diff
@@ -112,3 +112,235 @@ body {
     width: 30%;
   }
```

Comment:
- [ ] Currently, you have Inter font declared in most of your css, butt you haven't imported it from google yet. Kindly import "Inter" font from google to make your fonts work. [This is the link](https://fonts.google.com/share?selection.family=Inter:wght@200;300;400;500;600;700) to inter fonts on google. 
- You can use it the same way you imported poppins into your css file. see an example below:

## EXAMPLE 
```  
@import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700&display=swap');
```